packages/elements/src/slider/elements/slider.ts

```diff
@@ -1263,7 +1273,7 @@ export class Slider extends FormFieldElement {
    * @returns {void}
    */
   private onFromValueChange(): void {
-    if (this.isValueInBoundary(this.fromNumber, SliderDataName.from)) {
+    if (this.isValueInBoundary(this.fromNumber + this.minRangeNumber, SliderDataName.from)) {
```

Comment:
Can you move this logic to `isValueInBoundary`? Do that to both onFromChange and onTochange.

Reply:
I mean move only logic of the new change.